mods/ts/rules/civilian-infantry.yaml

```diff
@@ -270,13 +272,16 @@ VISC_LRG:
 		HP: 500
 	Armament:
 		Weapon: SlimeAttack
+		FireDelay: 10
```

Comment:
Are you sure you selected Creeps as owner and the Multi players as enemies?

They don't attack on Tiberium Garden Redux for example because the Owner is set to Neutral (and yes, changing that to Creeps worked for me).

Either way, I guess I should fix the official maps with this PR.